### ISSUE TITLE
custom default_language filepath

### ISSUE THREAD

**Issue by LazerJesus** [max-niederman/ttyper#133]
super cool tool. 
I use it for daily practice on custom files to practice syntax ( {}[];'\:"|,./<>?).
is there a way to add my files to the resources and maybe even set my custom file as a `default_language`?

**Comment by philocalyst** [comment]
yes! There is! You can define your word list as a plaintext file, with each word on its own line.

From my personal config:
`default_language = "/Users/philocalyst/Projects/words.txt"`
